index.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable linebreak-style */
+/* eslint-disable func-names */
+/* eslint-disable semi */
+/* eslint-disable linebreak-style */
+/* eslint-disable import/extensions */
```

Comment:
The objective of using linters is to analyze source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Kindly remove these lines of code and fix the linter errors. :pray: :pray: